admin-api-server/test/coverage/summary.txt

```diff
@@ -1,164 +1,168 @@
 
---------------------------|---------|----------|---------|---------|---------------------------------------------------------------------------------------------
+--------------------------|---------|----------|---------|---------|-------------------------------------------------------------------------------
 
-File                      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                                                                           
+File                      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                                                             
 
---------------------------|---------|----------|---------|---------|---------------------------------------------------------------------------------------------
+--------------------------|---------|----------|---------|---------|-------------------------------------------------------------------------------
 
-All files                 |   84.83 |    83.43 |   84.96 |   84.83 |                                                                                             
+All files                 |   17.55 |    43.47 |   24.32 |   17.55 |                                                                               
```

Comment:
This large dropoff in covered code line percentage indicates that the tests eeither didn't succeed or not all were ran (because of selectively targeting a subset of the tests) the last time you ran them (and committed this coverage file into git), can you rerun to verify they pass, and commit the changed coverage report?

Reply:
Yes I forgot to run after rebase, now it is fixed